### ISSUE TITLE
Generate well formed http request

### ISSUE THREAD

**Issue by sts** [matteocorti/check_ssl_cert#100]
It would be great if check_ssl_cert could generate a well formed http/1.1 request for the checks as it clutters our haproxy logs with Status `400`/`BADREQ` errors:

Sample http request

```http
HEAD / HTTP/1.1
Host: example.com
User-Agent: check_ssl_cert/1.73.0
Connection: close
```

Something along the lines should do, although I think users would even want to configure the request for certain edge cases:

```bash

if [ -n "${SNI}" ]; then
            HOST_HEADER="${SNI}"
else
            HOST_HEADER="${HOST}"
fi


HTTP_REQUEST="HEAD / HTTP/1.1\nHost: ${HOST_HEADER}\nUser-Agent: check_ssl_cert/${VERSION}\nConnection: close\n\n"
exec_with_timeout "$TIMEOUT" "echo $HTTP_REQUEST | $OPENSSL s_client ${CLIENT} ${CLIENTPASS} -crlf -ign_eof -connect $HOST:$PORT ${SERVERNAME} -showcerts -verify 6 ${ROOT_CA} ${SSL_VERSION} ${SSL_VERSION_DISABLED} ${SSL_AU} 2> ${ERROR} 1> ${CERT}"```
```

**Comment by matteocorti** [comment]
Thanks for the suggestion. It definitely makes sense.

**Comment by matteocorti** [comment]
Hi,
How are you performing the check? Which web server are you using?
Apache 2.4 does not give me any errors.
Matteo

**Comment by matteocorti** [comment]
Fixed in 1.76.0
